pkg/remoteconfig/state/agent_config.go

```diff
@@ -26,7 +26,8 @@ type AgentConfig struct {
 
 // ConfigContent contains the configurations set by remote-config
 type ConfigContent struct {
-	LogLevel string `json:"log_level"`
+	LogLevel         string `json:"log_level"`
+	EnableStreamLogs bool
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏You'll also need to populate it next to the log level
https://github.com/DataDog/datadog-agent/pull/25915/files#diff-199ee81718422b82a5e067222fba86cc8f18d4c5354a4436e9e61e77e008648fL148 and https://github.com/DataDog/datadog-agent/pull/25915/files#diff-199ee81718422b82a5e067222fba86cc8f18d4c5354a4436e9e61e77e008648fL151

You might also want to make it `*bool` so you can differentiate `false` from `unset`